### PR TITLE
Add unit tests for public API

### PR DESCRIPTION
**Pull request by dongheeleeme**
I've added unit tests for public API(XFloat, XInt, YFloat, YInt) based on [this issue](https://github.com/cashapp/contour/issues/128). Do we have some conventions to write unit tests? If so, I'll make changes according to the rules. Thanks 😄

### CODE REVIEW

File: contour/src/test/kotlin/com/squareup/contour/XFloatTest.kt [hunk 1]
```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 Square Inc.
```
Comment (oldergod) [inline contour/src/test/kotlin/com/squareup/contour/XFloatTest.kt:2]:
```suggestion
 * Copyright 2021 Square Inc.
```

File: contour/src/test/kotlin/com/squareup/contour/XIntTest.kt [hunk 2]
```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 Square Inc.
```
Comment (oldergod) [inline contour/src/test/kotlin/com/squareup/contour/XIntTest.kt:2]:
```suggestion
 * Copyright 2021 Square Inc.
```

File: contour/src/test/kotlin/com/squareup/contour/YFloatTest.kt [hunk 3]
```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 Square Inc.
```
Comment (oldergod) [inline contour/src/test/kotlin/com/squareup/contour/YFloatTest.kt:2]:
```suggestion
 * Copyright 2021 Square Inc.
```

File: contour/src/test/kotlin/com/squareup/contour/YIntTest.kt [hunk 4]
```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 Square Inc.
```
Comment (oldergod) [inline contour/src/test/kotlin/com/squareup/contour/YIntTest.kt:2]:
```suggestion
 * Copyright 2021 Square Inc.
```

**Review by oldergod — APPROVED** [review]
Amazing job, thank you.

Could you sign our [CLA](https://docs.google.com/forms/d/e/1FAIpQLSeRVQ35-gq2vdSxD1kdh7CJwRdjmUA0EZ9gRXaWYoUeKPZEQQ/viewform?formkey=dDViT2xzUHAwRkI3X3k5Z0lQM091OGc6MQ&ndplr=1) so I can merge your PR?

**Comment by oldergod** [comment]
fixes #128

**Comment by dongheeleeme** [comment]
> Amazing job, thank you.
> 
> Could you sign our [CLA](https://docs.google.com/forms/d/e/1FAIpQLSeRVQ35-gq2vdSxD1kdh7CJwRdjmUA0EZ9gRXaWYoUeKPZEQQ/viewform?formkey=dDViT2xzUHAwRkI3X3k5Z0lQM091OGc6MQ&ndplr=1) so I can merge your PR?

Awesome, thank you so much 😄 
I just signed CLA form.

File: contour/src/test/kotlin/com/squareup/contour/YIntTest.kt (hunk 4 above)
Comment (dongheeleeme) [reply contour/src/test/kotlin/com/squareup/contour/YIntTest.kt]: Updated in 923db8f

File: contour/src/test/kotlin/com/squareup/contour/YFloatTest.kt (hunk 3 above)
Comment (dongheeleeme) [reply contour/src/test/kotlin/com/squareup/contour/YFloatTest.kt]: Updated in 923db8f

**Comment by oldergod** [comment]
Thanks!

**Pull request merged**